test/Project.toml

```
@@ -19,5 +19,5 @@ ForwardDiff = "0.10"
 MPI = "0.16, 0.17, 0.18"
 OrdinaryDiffEq = "5.44 - 5.50, 5.51.1"
 Plots = "1.9"
-StartUpDG = "0.8"
+StartUpDG = "0.8, 0.9.1, 0.10"
```

```suggestion
StartUpDG = "0.10"
```
Some of your commit messages read like "We should use the new version of StartUpDG". Is that correct? If so, I would propose to just request the newest version here.

---

Sounds good. 

The flood of versions came b/c I broke the public API on accident...